src/cluecode/finder_data.py

```diff
@@ -223,10 +246,13 @@ def classify(s, data_set, suffixes=None):
 
 classify_host = partial(classify, data_set=JUNK_HOSTS_AND_DOMAINS, suffixes=JUNK_DOMAIN_SUFFIXES)
 
-classify_email = partial(classify, data_set=JUNK_EMAILS, suffixes=JUNK_DOMAIN_SUFFIXES)
+classify_email = partial(classify, data_set=JUNK_EMAILS, suffixes=JUNK_DOMAIN_SUFFIXES, ignored_hosts=JUNK_EXACT_DOMAIN_NAMES)
 
 
 def classify_url(url):
+    """
+    Return False if `url` found to be a candidate for a spam url. Return True otherwise.
```

Comment:
What about this instead?
```
Return True if `url` is a good url or False if this is spam url.
```
BUT since this is completely unrelated to your main PR, may be this addition is not needed?